README.md

```diff
@@ -20,6 +20,7 @@ This repo hosts samples for working with the OpenTelemetry Operator on GCP.
 * [Helm](https://helm.sh) (for GKE Autopilot)
```

Comment:
looks like you accidentally pulled in changes from https://github.com/GoogleCloudPlatform/opentelemetry-operator-sample/pull/31/commits/34ea308a0f5f7aad05e5ffb3d03fc9df155bf2c5 to this file, I recommend `git rebase` instead of pull/merge

Reply:
@damemi oh yeah. i created a new fresh PR.